example/spec/testAPI/APISuccessSpec.js

```diff
@@ -239,4 +239,28 @@ describe('The Cumulus API', () => {
       });
     });
   });
+
+  describe('providers endpoint', () => {
```

Comment:
Since these tests aren't really connected to the workflow execution in this spec, I'm wondering if they would be better handled as unit tests? If so, we already have unit tests for lists of providers/collections, but would need to add one for workflows